vllm/v1/attention/backends/pallas.py

```diff
@@ -214,8 +215,8 @@ def write_to_kv_cache(
 
     kv = torch.cat([key, value], axis=-1).reshape(-1, num_combined_kv_heads,
                                                   head_size)
-
-    torch.ops.xla.dynamo_set_buffer_donor_(kv_cache, True)
+    if os.environ.get("VLLM_TORCHAX_ENABLED", "0") == "0":
```

Comment:
Please add the environment variable to https://github.com/vllm-project/vllm/blob/aea302be6c3c323207502a973fe341c3bcf7288f/vllm/envs.py instead of using os.environ directly